gephi/gephi-plugins
# Spatial Ranking comeback

## New plugin or plugin update?


## What is the purpose of this plugin?

Map attributes to X/Y/Z coordinates to create scatter plots.

Requested in #271 by @grandjeanmartin.

## How to test your plugin in Gephi?

1. Load any graph, preferably with some columns
2. Go to Appearance and notice the extra icon for coordinates
3. Select Ranking and the column you want to map to coordinates
4. Select the axis and the min/max size
5. Press Apply

Note that I also added a "Simple" transformer that does the same thing as the random layout.

## Checklist before submission

* [X] Did you write unit tests to test your plugin functionalities? -> Yes added 2 basic tests
